tex/gregoriotex-signs.tex

```diff
@@ -1726,118 +1732,123 @@
   \gre@newglyphcommon %
   \gre@calculate@glyphraisevalue{\gre@pitch@bar}{0}{}% bar glyphs are made to be at this height
   \GreNoBreak %
+  % count indicating if we have to output space or not:
+  \gre@count@temp@one=0\relax %
+  \ifnum#2=1\relax %
+    \gre@count@temp@one=1\relax %
+  \else %
+    \ifgre@newbarspacing %
+      \gre@count@temp@one=1\relax %
+    \fi %
+  \fi %
   \ifcase#1 % 0 : virgula
-    \ifnum#2=1\relax %
-      \gre@skip@temp@four = \csname gre@space@skip@spacearoundsmallbar\gre@bar@text{#3}\endcsname\relax%
+    \ifnum\gre@count@temp@one=1\relax %
+      \gre@skip@temp@four = \csname gre@space@skip@bar@virgula\gre@bar@space@suffix{#2}{#3}\endcsname\relax%
       \gre@hskip\gre@skip@temp@four %
```

Comment:
The stretchability of these skips is creating a problem in the new bar spacing algorithm because it assumes that the box containing a bar has a fixed size, but with these stretchable skips inside it doesn't.  Can these be changed to non-stretchable dimensions?